packages/pigment-css-utils/tests/utils/processStyle.test.ts

```diff
@@ -0,0 +1,167 @@
+import { expect } from 'chai';
+import { processStyle, processStyleObjects } from '../../src/utils';
+
+describe('processStyle', () => {
+  describe('processStyle', () => {
+    it('should transform css variables', () => {
+      let count = 0;
+      function getVariableName() {
+        count += 1;
+        return `var-${count}`;
+      }
+      const style = {
+        $$hello: 'world',
+        $hello1: 'world',
```

Comment:
```suggestion
        $$hello: 'world',
        $hello1: 'world',
```

What's the difference betweem $ and $$?